source/development/coredump.rst

```diff
@@ -117,3 +117,46 @@ On macOS, most applications have core dump generation disabled by default. Howev
       # sysctl -w kern.corefile=/cores/core.%P
 
 Enabling core dump generation might consume significant disk space, so use it judiciously. Moreover, not all processes on macOS support or behave consistently with core dump generation.
+
+Windows endpoints
+-----------------
+    .. note:: Remember to be cautious when modifying the Windows Registry. Incorrect changes can cause system instability or other issues. Always backup the registry or create a system restore point before making changes.
+
+To collect user-mode dumps on Windows, you can use the built-in Windows Error Reporting (WER) feature. Here's how you can set it up:
+
+#. **Windows + R** keys on your keyboard to open the run dialog box.
+
+#. Type **regedit** in the search box and click **OK** to open the registry editor.
+
+#. Navigate to the Registry Key:
+
+   .. code-block:: none
+
+        HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps
+
+   .. note:: If the ``LocalDumps`` key doesn't exist, you may need to create it.
+
```

Comment:
```suggestion
Configuring Windows Error Reporting
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

#. Navigate to the ``LocalDumps`` registry key or create it, as it might not exist by default.

   .. code-block:: none

      HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps

```